test/e2e/specs/todomvc.js

```diff
@@ -105,7 +105,7 @@ module.exports = {
       .dblClick('.todo:nth-child(1) label')
       .assert.count('.todo.editing', 1)
       .assert.focused('.todo:nth-child(1) .edit')
-      .clearValue('.todo:nth-child(1) .edit')
+    deleteValue('.todo:nth-child(1) .edit', 'test2')
```

Comment:
`clearValue` was not working. Not sure why but after `clearValue`, 2 todos was removed before `setValue`. So I'm guessing `clearValue` is emitting `blur` event or something...

So, I went ahead and created `deleteValue` function which will input "backspace" and deletes value. If this is the only workaround we have at the moment, I would like to extract that into a custom command so that we can do `browser.deleteValue`. Ref: [Custom Command Doc](https://nightwatchjs.org/guide/extending-nightwatch/#writing-custom-commands)